src/analyses/variable-sensitivity/abstract_object.h

```diff
@@ -258,6 +258,9 @@ class abstract_objectt : public std::enable_shared_from_this<abstract_objectt>
     abstract_object_pointert op1,
     abstract_object_pointert op2,
     bool &out_modifications);
+  static abstract_object_pointert merge(
+    abstract_object_pointert op1,
+    abstract_object_pointert op2);
```

Comment:
How about changing the interface to make `merge` return a pair?

Reply:
I did try it, but it didn't really make the code any clearer. If structured bindings were available, I'd absolutely have gone for it.